UnityProject/Assets/Scripts/Effects/Shake.cs

```diff
@@ -1,53 +1,51 @@
 ﻿using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
+using Mirror;
 
-public class Shake : MonoBehaviour
+public class Shake : LTEffect
 {
     /// <summary>
-    /// A shake effect that comes in two modes, SPRITE and GAMEOBJECT
-    /// SPRITE animates a given sprite only.
-    /// GAMEOBJECT animates the entire object's position.
+    /// A shake effect that shakes an entire GameObject or it's sprites only.
```

Comment:
what's the benefit of being able to choose between object and sprite?

Reply:
If an object doesn't have a sprite handler or has multiple objects that need to shake GAMEOBJECT mode makes sure that the object will be shaken correctly, there's also some gameplay benefits like synced hitboxes and stuff like that.
GAMEOBJECT mode is there for those "what if" moments so people in the future don't have to worry about coding it themselves or even worse coding their own solution outside a script that already exists for it.